01/userinfamous/wordvalue.py

```diff
@@ -0,0 +1,33 @@
+from data import LETTER_SCORES, DICTIONARY 
+import os
+
+#File path for Dictionary
+fname = os.path.join('../',DICTIONARY)
+
+#Wordlist from dictionary
+def load_words() -> list:
+    with open(fname,'r') as f:
+        wordlist = f.read().split()
+        return wordlist
+
+def calc_word_value(word:str) -> int:
+    score = 0
+    for letter in word:
+        if letter.upper() in LETTER_SCORES:
+            score += LETTER_SCORES[letter.upper()]
```

Comment:
try to use list comprehensions: `score = sum(LETTER_SCORES.get(letter.upper(), 0) for letter in word)`. Normally we would have prooved `if letter.upper() in LETER_SCORES` but here we can use the convenient `get` method of dictionaries